### DISCUSSION TITLE
Works on Spring 2.3 and 2.4?

### DISCUSSION THREAD

**Discussion by netstart** [Cosium/spring-data-jpa-entity-graph#55] [General]
For the first, congratulation for this implementation. I'd like see this in a oficial Spring framework repository.

Now, my question...

Is this Works on Spring 2.3.x and 2.4.x ?

**Comment by reda-alaoui** [comment]
Hello,

Doesn't the compatibility matrix (https://github.com/Cosium/spring-data-jpa-entity-graph#documentation) of `README.md` answer this question?
